src/init/prepare.sh

```diff
@@ -272,6 +274,23 @@ elabimg: info: runtime configuration successfully finished
 EOT
 }
 
+migrations() {
```

Comment:
I think it would be best to split this in two different functions, one for init, one for update. Also documenting these options in the example config file is necessary.